nixos/modules/profiles/macos-builder.nix

```
@@ -133,6 +133,7 @@ in
         enable = true;
 
         authorizedKeysFiles = [ "${keysDirectory}/%u_${keyType}.pub" ];
+        authorizedKeysInHomedir = false;  # working around #325610
```

I'm inclined to just drop this, since the VM is not dependent on SSH to change the config, and it's supposed to be minimal.

```suggestion
        authorizedKeysInHomedir = false;
```

@Gabriella439 do you agree?